src/controllers/hid/legacyhidcontrollermappingfilehandler.h

```diff
@@ -1,5 +1,7 @@
 #pragma once
 
+#include <gtest/gtest_prod.h>
```

Comment:
This will cause a build failure when gtest is not available. 
Subject of: https://github.com/mixxxdj/mixxx/pull/15736 
Relevant for flatpack. 
How about to use the proposed guard here to avoid conflict in a distance later.